docs/src/assets/scripts.js

```diff
@@ -55,6 +72,23 @@ const establishDisplay = () => {
       otpCol.style.display = 'none';
       otpBtn.style.display = 'block';
     }
+    const prev = localStorage.getItem('otp-col');
+    switch (prev) {
+      case 'block':
+        otpCol.style.display = 'block';
+        otpBtn.style.display = 'none';
+        break;
+
+      case 'none':
+        otpCol.style.display = 'none';
+        otpBtn.style.display = 'block';
+        break;
+
+      default:
+        const { display } = otpCol.style;
+        localStorage.setItem('otp-col', display);
+        break;
+    }
```

Comment:
Capture these two code changes into one function that you call `functionWithGreatName('otp-col', otpCol, otpBtn)`... even better, have this function combine the common code that these are both part of (that looks at the widths)